docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,120 @@
+```
+
+### Setting up Etcd-Druid
+
+Generates the Etcd CRD and deploy an etcd-druid pod into the cluster
+
+```sh
+make deploy
+```
+
+### Applying the etcd CR
+
+- **Without Backups enabled**
+
+    To setup Etcd-druid without backups enabled, apply the Etcd yaml to the cluster
+
+    Confirm that the `spec.backup.store` is commented out.
+
+    ```sh
+    # Apply the etcd CR yaml
+    kubectl apply -f config/samples/druid_v1alpha1_etcd.yaml
+    ```
+
+- **With Backups enabled (On Cloud Provider Object Stores)**
+
+    Create a secret for cloud provider access. Find the secret yaml templates for different cloud providers [here](https://github.com/gardener/etcd-backup-restore/tree/master/example/storage-provider-secrets).
+
+    Replace the dummy values with the actual configurations and make sure to add a name and a namespace to the secret as intended.
+
+    > **Note 1:** The secret should be applied in the same namespace as druid.
+    >
+    > **Note 2:** All the values in the data field of secret yaml should be in base64 encoded format.
+
+    **Apply the secret**
+
+    ```sh
+    kubectl apply -f path/to/secret
+    ``````
+
+    Uncomment the `spec.backup.store` section of the druid yaml and set the keys to allow backuprestore to take backups by connecting to an object store.
+
+    ```yaml
+    # Configuration for storage provider
+    store:
+        secretRef:
+            name: etcd-backup-secret-name
+        container: object-storage-container-name
+        provider: aws # options: aws,azure,gcp,openstack,alicloud,dell,openshift,local
+        prefix: etcd-test
+    ```
+
+    Brief explanation of keys:
+
+  - `secretRef.name` is the name of the secret that was applied as mentioned above
+  - `store.container` is the object storage bucket name
+  - `store.provider`  is the bucket provider. Pick from the options mentioned in comment
+  - `store.prefix`    is the folder name that you want to use for your snapshots.
+
+    > **Note:** Before applying the Etcd yaml, make sure the bucket is created in the corresponding cloud provider
+
+    Create the Etcd CR (Custom Resource) by applying the Etcd yaml to the cluster
+
+    ```sh
+    # Apply the etcd CR yaml
+    kubectl apply -f config/samples/druid_v1alpha1_etcd.yaml
+    ```
+
+### Annotate the etcd CR
+
+The above step creates an Etcd resource, however etcd-druid won't pick it up for reconciliation without an annotation. To get etcd-druid to reconcile the etcd CR, annotate it with the following `gardener.cloud/operation:reconcile`.
+
+```sh
+# Annotate etcd-test CR to reconcile
+kubectl annotate etcd etcd-test gardener.cloud/operation="reconcile"
+```
+
+This starts creating the etcd cluster
+
+## Cleaning the setup
+
+```sh
+# Delete the cluster
+make kind-down
+```
+
+This cleans up the cluster i.e deletes the created Etcd and all pods that got created along the way and also other resources such as statefulsets, services, PV's, PVC's, etc.
```

Comment:
This says that the etcd cluster is cleared, but is not clear about the kind cluster being deleted. Can you please update this so that it's clear the kind cluster is deleted?